cmd/end2endtest/account.go

```diff
@@ -46,14 +46,13 @@ func (t *E2ETokenTxs) Setup(api *apiclient.HTTPclient, config *config) error {
 	t.alice = ethereum.NewSignKeys()
 	err := t.alice.Generate()
 	if err != nil {
-		return err
+		return fmt.Errorf("error in generate new alice keys: %w", err)
 	}
 
 	// create bob signer
 	t.bob = ethereum.NewSignKeys()
-	err = t.bob.Generate()
-	if err != nil {
-		return err
+	if err := t.bob.Generate(); err != nil {
+		return fmt.Errorf("cannot generate new bob keys: %w", err)
```

Comment:
keep the consistency:

* either inline `if blah ; err != nil { }` (as in bob) , or two separate lines (as in alice).

* keep the error strings as consistent as possible. why is alice `error in generate` and bob `cannot generate`?
* since this is a test tool and those strings are never exposed to any enduser, i'd even suggest to use "code-related" strings, as in `fmt.Errorf("bob.Generate: %w", err)` and so on, like you did with testGetTxCost. 
* if you want to be more verbose than that (for example `fmt.Errorf("error in bob.Generate: %w", err)`, please keep the consistency, always use `error in` instead of `cannot retrieve`, `failed to`, and all the variations. The string is already unique because of the keyword (testGetTxCost, etc).